data/sql/updates/pending_db_world/rev_1580747652284896800.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1580747652284896800');
+DELETE FROM `command` WHERE  `name`='demorph';
+INSERT INTO `command` (`name`, `security`, `help`) VALUES ('morph target', '1', 'Syntax: .morph target #displayid - Change the target\'s current model id to #displayid.');
```

Comment:
I think it's better to use two single quotes `''` instead of backslash and single quote `/'` in order to escape the single quote:

From: `Change the target\'s `
To: `Change the target''s `

while the first would probably work almost everywhere, it's not the official SQL syntax (and the github PR preview is also a bit broken, probably because of that)